### ISSUE TITLE
Update and design accessibility statement

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#700]
We would like to review the accessibility to determine if it needs update or more content design and make those changes if necessary

## Description

As a public servant using Notify, I need to understand what steps Notify has taken toward accessibility so that I can feel confident in the product and understand how to raise accessibility issues. 

WHY are we building?
To provide our users with information about the accessibility of Notify.
CDS unpublished its a11y handbook. We could update it and make our own, and not call it a handbook

WHAT are we building?
An up to date and easy to read and scan accessibility statement, including parts of the CDS a11y handbook that are relevant to share.

VALUE created by our solution
We build our users confidence in our commitment to accessibility and our willingness to address new issues that arise. 

## Acceptance Criteria** (Definition of done) 

_To be refined through discussion with the team_

Given I need the service to be accessible, when I want information about Notify's accessibility, then I can easily find the accessibility statement and the statement answers my questions.

If this user story emerged from User Research insights:
- [ ] Link research insight back to hypothesis from Epic or Objective in Airtable
- [ ] Once change/fix/feature is implemented, mark insight as "resolved" in Airtable
- [ ] Once change/fix/feature is implements, link insight to design artifacts (Figma) in Airtable
- [ ] Check that "Known issues and limitations section" is current

* A11y
* Measuring success and metrics

## QA Steps

- [ ] Tested in a realistic production scenario

**Comment by yaelberger-commits** [comment]
Likely related to issue #702

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/700?planning-poker) with Zenhub @amazingphilippe @adriannelee @YedidaZalik @sharlychan-cds

**Comment by YedidaZalik** [comment]
From UX Content Collective course, Accessibility for UX Writers and Designers, Lesson 2.2:

Publish an accessibility statement
In addition to understanding WCAG and applying it in your role, you can create an accessibility statement to let people know your company cares about accessibility. As defined by WAI, this is a page that states:

A commitment to accessibility for people with disabilities
The standard applied, such as WCAG 2.2 level AA
Contact information in case people encounter problems
Make sure your company has a process in place for handling accessibility feedback and support.

WAI also advises that you include the following information:

Any known limitations, to avoid frustration of your users
Measures taken by your organization to ensure accessibility
Technical prerequisites, such as supported web browsers
Environments in which the content has been tested
References to applicable national or local laws and policies
Accessibility statements should be easy to find. You can link to them from the site footer, help menu, about menu, and other prominent areas.

Intuit footer with About Intuit, Join Our Team, Press Room, Accessibility, and Terms and Conditions links.
Accessibility link in the Intuit site footer.

The WAI site is a great resource for creating your statement with templates for minimal and complete versions. Here are some examples from the web:
AbilityNet
Australian Broadcasting Corporation
eBay
Intuit
LinkedIn
Nature
Unilever
